### ISSUE TITLE
sbert发现一处bug？

### ISSUE THREAD

**Issue by Garrfiiel** [Macielyoung/sentence_representation_matching#4]
sbert/train.py compute_acc()函数 23行accuracy_score() pred和lebel入参好像反了？
`acc = accuracy_score(pred.detach().cpu().numpy(), label.cpu().numpy())`
应该是`acc = accuracy_score(label.cpu().numpy()， pred.detach().cpu().numpy())`吗？

![截图 2023-12-02 00-55-36](https://github.com/Macielyoung/sentence_representation_matching/assets/55494106/0fa29f31-3454-4740-be2e-935ec9d0684e)

**Comment by Macielyoung** [comment]
感谢指正🤝
